napari/_qt/qt_main_window.py

```diff
@@ -1524,16 +1529,43 @@ def _screenshot(
             If True, screenshot shows only the image display canvas, and
             if False include the napari viewer frame in the screenshot,
             By default, True.
+        fit_to_data: bool
+            Whether to fit a bounding box around the data to prevent margins of showing in the screenshot.
+            Currently, if this is False it means a screenshot of the current canvas will be generated with
+            black margins if visible.
```

Comment:
I think the "bounding box" talk is a bit confusing.

```suggestion
            Tightly fit the canvas around the data to prevent margins of showing in the screenshot.
            If False, a screenshot of the whole currently visible canvas will be generated.
```